ci/gpu/build.sh

```diff
@@ -166,16 +166,28 @@ else
     gpuci_logger "Check GPU usage"
     nvidia-smi
 
-    gpuci_logger "GoogleTests"
     set -x
     cd $LIB_BUILD_DIR
 
+    gpuci_logger "GoogleTests"
+
     for gt in gtests/* ; do
         test_name=$(basename ${gt})
         echo "Running GoogleTest $test_name"
         ${gt} --gtest_output=xml:"$WORKSPACE/test-results/"
     done
 
+    ls -l "$WORKSPACE/test-results/"
```

Comment:
Is this for debug?

Reply:
Oops. Yes. This whole file is debug right now. :-)